gateways/paypal_pro/includes/paypal.nvp.class.php

```diff
@@ -72,6 +73,7 @@ function Espresso_PayPal($DataArray)
 		$this -> NVPCredentials .= $this -> APISubject != '' ? 'SUBJECT=' . $this -> APISubject : '';
 		$this -> NVPCredentials .= $this -> APIMode == 'Signature' ? '&SIGNATURE=' . $this -> APISignature : '';
 
+		var_dump($this->NVPCredentials);
```

Comment:
OOPS!

going to go out on a limb here and say it's maybe not a good thing to dump credentials like this 🤔